### DISCUSSION TITLE
Pause Menu Shortcut CTRL + Q doesnt work in Crosshair Mode

### DISCUSSION THREAD

**Discussion by ihategravel2** [Mojang/minecraft-editor#14] [Bugs]
You can use the shortcut CTRL + Q to get to the pause screen in editor mode but the same shortcut doesn't do anything when your in crosshair mode.

**Comment by minecraftchloe** [comment]
This is expected functionality and by design for keybinding requirements in Tool vs Crosshair Modes.

**Reply by ihategravel2** [reply]
Okay, thanks!
